content/applications/finance/expenses.rst

```diff
@@ -191,104 +229,137 @@ To submit an expense via email, create a new email and enter the product's *inte
+to an individual expense, a :guilabel:`📎(paperclip)` icon appears next to the :guilabel:`Customer
+to Reinvoice` and :guilabel:`Analytic Distribution` columns.
+
+When the report is created, the date range for the expenses appears in the :guilabel:`Expense Report
+Summary` field by default. It is recommended to edit this field with a short summary for each report
+to help keep expenses organized. Enter a short description for the expense report (such as `Client
+Trip NYC`, or `Repairs for Company Car`) in the :guilabel:`Expense Report Summary` field. Next,
+select a :guilabel:`Manager` from the drop-down menu to assign a manager to review the report. If
+needed, the :guilabel:`Journal` can be changed. Use the drop-down menu to select a different
+:guilabel:`Journal`.
 
 .. image:: expenses/expense-report-summary.png
    :align: center
    :alt: Enter a short description and select a manager for the report.
 
 If some expenses are not on the report that should be, they can still be added. Click :guilabel:`Add
-a line` at the bottom of the :guilabel:`Expense` tab. Click the check box next to each expense to
-add, then click :guilabel:`Select`. The items now appear on the report that was just created.
+a line` at the bottom of the :guilabel:`Expense` tab. A pop up appears with all the available
+expenses that can be added to the report (a status of :guilabel:`To Submit`). Click the check box
+next to each expense to add, then click :guilabel:`Select`. The items now appear on the report that
+was just created. If a new expense needs to be added that does *not* appear on the list, click
+:guilabel:`New` to create a new expense and add it to the report.
 
 .. image:: expenses/add-an-expense-line.png
    :align: center
    :alt: Add more expenses to the report before submitting.
 
 .. note::
-   :guilabel:`Add a line` only appears when the document is in edit mode. It does not appear
-   otherwise.
+   Expense reports can be created in one of three places:
 
-When all edits have been completed, click :guilabel:`Save`.
+   #. Go to the main :menuselection:`Expenses` app dashboard (also accessed by going to
+      :menuselection:`Expenses app --> My Expenses --> My Expenses`)
+   #. Go to :menuselection:`Expenses app --> My Expenses --> My Reports`
+   #. Go to :menuselection:`Expenses app --> Expense Reports`
+
+   In any of these views, click :guilabel:`New` to create a new expense report.
+
+.. _expenses/submit:
 
 Submit an expense report
 ------------------------
 
 When an expense report is completed, the next step is to submit the report to a manager for
 approval. Reports must be individually submitted, and cannot be submitted in batches. Open the
 specific report from the list of expense reports (if the report is not already open). To view all
-expense reports, go to :menuselection:`Expenses app --> My Expenses --> My Reports`.
+your expense reports, go to :menuselection:`Expenses app --> My Expenses --> My Reports`.
```

Comment:
My logic was, that when you go to "my" reports, it is only "your "reports- but if you look at "all" reports (a different filter) and you are a manager, you see ALL reports- reports for other people you can approve, not just yours. So I thought it might be important to mention it was only "your" reports- even though I know that word does not belong in the doc .  But I took it out- hopefully people get it on their own, or from context =)